src/element.financial.js

```diff
@@ -0,0 +1,109 @@
+﻿'use strict';
+
+module.exports = function(Chart) {
+
+	var globalOpts = Chart.defaults.global;
+
+	globalOpts.elements.financial = {
+		color: {
+			up: "rgba(80, 160, 115, 1)",
+			down: "rgba(215, 85, 65, 1)",
+			linear: "rgba(90, 90, 90, 1)",
```

Comment:
`linear` is confusing to me. is it supposed to represent something like `border` or `unchanged`?

Reply:
Renamed to 'unchanged'.